File: 백준/Bronze/16676. 근우의 다이어리 꾸미기/근우의 다이어리 꾸미기.c
#include <stdio.h>

int main() {
    int std = 11;
    int dap = 1;
    int N;
    scanf("%d", &N);
    while (1) {
        if (N < std) {
            printf("%d\n", dap);
            break;
        } else {
            std = std * 10 + 1;
            dap += 1;
        }
    }
    return 0;
}

File: 백준/Bronze/16676. 근우의 다이어리 꾸미기/근우의 다이어리 꾸미기.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        int std = 11;
        int dap = 1;
        Scanner sc = new Scanner(System.in);
        int N = Integer.parseInt(sc.nextLine());
        while (true) {
            if (N < std) {
                System.out.println(dap);
                break;
            } else {
                std = std * 10 + 1;
                dap += 1;
            }
        }
        sc.close();
    }
}